supporter-product-data/cloudformation/cfn.yaml

```diff
@@ -0,0 +1,383 @@
+                "Next": "WaitSomeTime",
+                "Retry": [
+                   {
+                    "ErrorEquals": ["States.ALL"],
+                    "IntervalSeconds": 30,
+                    "MaxAttempts": 3
+                  }]
+              },
+              "WaitSomeTime": {
+                "Type": "Wait",
+                "Seconds": 30,
+                "Next": "FetchResults"
+              },
+              "FetchResults": {
+                "Type": "Task",
+                "Resource": "${fetcherArn}",
+                "Next": "CheckForNewSubscriptions",
+                "Retry": [
+                   {
+                    "ErrorEquals": ["States.ALL"],
+                    "IntervalSeconds": 60,
+                    "MaxAttempts": 20,
+                    "BackoffRate": 1.0
+                  }]
+              },
+              "CheckForNewSubscriptions": {
+                "Type": "Choice",
+                "Choices": [
+                  {
+                      "Variable": "$.recordCount",
+                      "NumericEquals": 0,
+                      "Next": "NoNewSubscriptions"
+                  }
+                ],
+                "Default": "UpdateDynamo"
+              },
+              "NoNewSubscriptions": {
+                "Type": "Pass",
+                "End": true
+              },
+              "UpdateDynamo": {
+                "Type": "Task",
+                "Resource": "${dynamoArn}",
+                "Next": "CheckProcessingComplete",
+                "Retry": [
+                   {
+                    "ErrorEquals": ["States.ALL"],
+                    "IntervalSeconds": 30,
+                    "MaxAttempts": 3,
+                    "BackoffRate": 1.0
+                  }]
+              },
+              "CheckProcessingComplete": {
+                "Type": "Choice",
+                "Choices": [
+                  {
+                      "Variable": "$.recordCount",
+                      "NumericGreaterThanPath": "$.processedCount",
+                      "Next": "UpdateDynamo"
+                  }
+                ],
+                "Default": "ProcessingComplete"
+              },
+              "ProcessingComplete": {
+                "Type": "Pass",
+                "End": true
+              }
+            }
+          }
+        - {
+          querierArn: !GetAtt [ QueryZuoraLambda, Arn ],
+          fetcherArn: !GetAtt [ FetchResultsLambda, Arn ],
+          dynamoArn: !GetAtt [ UpdateDynamoLambda, Arn ]
+        }
+      RoleArn: !GetAtt [ StatesExecutionRole, Arn ]
+
+  SupporterProductDataScheduledRule:
+    Type: "AWS::Events::Rule"
+    Properties:
+      Description: "Trigger regular data synchronisation from Zuora"
+      ScheduleExpression: !FindInMap [StageVariables, !Ref Stage, scheduleRate]
+      State: "ENABLED"
+      Targets:
+        -
+          Arn: !Ref SupporterProductData
+          Id: !Sub trigger_state_machine-${Stage}-1
```

Comment:
why "-1" ?

Reply:
Copied from here and didn't notice: https://github.com/guardian/fulfilment-lambdas/blob/ba9cca4a2b36f7f9592a30b2a62a358b79e06644/cloudformation/cloudformation.yaml#L414

Reply:
Have updated